specification/gedcom-02-datatypes.md

```diff
@@ -317,6 +317,27 @@ where:
 
 The URI for the `MediaType` data type is `dcat:mediaType`.
 
+## Shortened URI
+
+A URI, possibly shorted and documented using a [documented extension tag](#extension-tags).
+
+```abnf
+ShortURI = extTag / URI
+```
+
+where:
+
+* `URI` is defined in [STD 66](https://www.rfc-editor.org/info/std66) section 3
+  as what is commonly called an "absolute URI"; notably, it always includes a colon.
+
+* Any `extTag` used shall be documented in the [schema] as a [documented extension tag](#extension-tags).
```

Comment:
make `[schema]` be a link?

Reply:
With the pandoc, the tool used to render this markdown, `[schema]` will be automatically interpreted as `[schema](#schema)`. I'm fine with replacing it with gfm-compatible markdown or leaving as is

Reply:
github doesn't render `[schema]` as a link so would recommend replacing it.